src/jsoncharutils_tables.h

```diff
@@ -201,8 +201,8 @@ const uint32_t digit_to_val32[886] = {
 // per instruction set).
 ///
 
-constexpr int FASTFLOAT_SMALLEST_POWER = -344;
-constexpr int FASTFLOAT_LARGEST_POWER = 308;
+constexpr int simdjson_smallest_power = -344;
```

Comment:
I'm actually OK with renaming all constants to lowercase--is that what you usually do?

I'm finding it looks a LOT better.

Reply:
"I'm actually OK with renaming all constants to lowercase--is that what you usually do?"

=> My expectation is that these were macros before that became constants...